### ISSUE TITLE
Pixelfed support

### ISSUE THREAD

**Issue by strk** [rugk/mastodon-simplified-federation#128]
The software does support remote interaction via ActivityPub.
See https://pixelfed.org/

**Comment by rugk** [comment]
Yeah, PRs welcome. Also an example page/workflow how the (remote) following button etc. looks like there would be helpfu.
